com.unity.render-pipelines.core/Runtime/Common/DynamicResolutionHandler.cs

```diff
@@ -421,8 +421,10 @@ public void Update(GlobalDynamicResolutionSettings settings, Action OnResolution
             if (!m_ForceSoftwareFallback && type == DynamicResolutionType.Hardware)
             {
                 hardwareResolutionChanged = FlushScalableBufferManagerState();
-                if (ScalableBufferManager.widthScaleFactor != m_PrevHWScaleWidth ||
-                    ScalableBufferManager.heightScaleFactor != m_PrevHWScaleHeight)
+                float sfw = ScalableBufferManager.widthScaleFactor;
```

Comment:
This change is a no-op right? I don't see the advantage as I tend to like more verbose naming vs acronyms, but am not opposed. However, if it is for clarity let me be an anal nitpicker and either we have _sfw_ and _sfh_ **or**  _wsf_ and _hsf_, now is a bit of a mixup of names.,

Reply:
This is left over on my side! i was debugging stuff. Good catch!